docs/spark-procedures.md

```diff
@@ -462,16 +462,29 @@ will then treat these files as if they are part of the set of files  owned by Ic
 
 #### Usage
 
-| Argument Name | Required? | Type | Description |
-|---------------|-----------|------|-------------|
-| `table`       | ✔️  | string | Table which will have files added to|
-| `source_table`| ✔️  | string | Table where files should come from, paths are also possible in the form of \`file_format\`.\`path\` |
-| `partition_filter`  | ️   | map<string, string> | A map of partitions in the source table to import from |
+| Argument Name | Required? | Type | Description                                                                                                                                            |
+|---------------|-----------|------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `table`       | ✔️  | string | Table which will have files added to                                                                                                                   |
+| `source_table`| ✔️  | string | Table where files should come from, paths are also possible in the form of \`file_format\`.\`path\`                                                    |
+| `partition_filter`  | ️   | map<string, string> | A map of partitions in the source table to import from                                                                                                 |
+| `check_duplicate_files`  | ️   | boolean | When true, will throw a exception if files added will result in duplicate (on by default, it's checking against files path in entries metadata table). |
```

Comment:
I think we should avoid implementation details here and also try to make it shorter for the table

How about?
```
Whether to prevent files existing in the table from being added.  (defaults to true)
```